standalone/Dockerfile

```diff
@@ -34,6 +34,28 @@ RUN git clone "$JAM_REPO" . --depth=1 --branch "$JAM_REPO_BRANCH" \
     && npm run build
 # --- UI builder - end
 
+
+# --- dinit builder
+FROM debian:bullseye-20221004-slim AS dinit-builder
+
+# install build dependencies
+RUN apt-get update \
+    && apt-get install -qq --no-install-recommends --no-install-suggests -y ca-certificates git gcc make m4 g++
+
+WORKDIR /usr/src/dinit
+
+# checkout and build project
+RUN git clone "https://github.com/davmac314/dinit" . --depth=1 --branch "v0.16.0" \
+    && chmod +x ./configs/mconfig.Linux.sh \
+    && ./configs/mconfig.Linux.sh \
+    && make \
+    && export ASAN_OPTIONS=detect_leaks=0 \
```

Comment:
I assume this disables leak detection, because otherwise `make check` will have a non-zero exit code which would break the `&&` chain?

Edit: Nvm, I found it, I think. Mentioned in the [BUILD docs](https://github.com/davmac314/dinit/blob/master/BUILD):

> The standard mconfig options enable various sanitizers during build of the tests. On Linux you may
see an error such as the following:
> [...]
> If you get this, either disable the address sanitizer or make sure you have overcommit enabled

Reply:
Exactly.

`make check` breaks with leak detection as the container would need to be started with `strace` capabilities, which might impact security. You can [see a failed build here](https://github.com/joinmarket-webui/jam-docker/actions/runs/3281963822/jobs/5404710886#step:7:5169), where leak detection was enabled.
> #34 1581.7 ==996==HINT: LeakSanitizer does not work under ptrace (strace, gdb, etc)

Disabled leak detection in favor of not running the tests at all.